src/test/java/org/zipcodewilmington/lab1/ArrayUtilities.java

```diff
@@ -0,0 +1,52 @@
+package org.zipcodewilmington.lab1;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Set;
+
+public class ArrayUtilities {
+
+    public static int[] moveLastToFirst(int[] values) {
+        {
+            if (values.length < 2)
+                return values;
+            int last = values[values.length - 1];
+
+            for (int i = values.length - 1; i > 0; i--) {
+                values[i] = values[i - 1];
+            }
+            values[0] = last;
+        }
+        return null;
+    }
+
+    public static int[] removeDuplicate(int[] values) {
+        int j = 0;
+        int i = 1;
+        if (values.length < 2)
+            return values;
+        while (i < values.length) {
```

Comment:
This looks clean. Good job. One nit picking advice is to name `i` and `j` to something more meaningful because when I get down to line 37, I'm not sure what is what anymore.